### ISSUE TITLE
[Bug]: Webhook.validateWebhook is not a function

### ISSUE THREAD

**Issue by magnafilix** [EasyPost/easypost-node#377]
### Software Version

6.4.0

### Language Version

node v18.15.0

### Operating System

macOS Monterey 12.6.5 (21G531)

### What happened?

1. installed easypost package (`npm i @easypost/api`)
2. created a class instance (`new Easypost(EASYPOST_API_KEY)`)
3. tried to use `validateWebhook` method (autocomplete suggests this method)
<img width="802" alt="236429371-edc6b5ad-2365-4009-9389-02b2adb9c170" src="https://user-images.githubusercontent.com/23529889/236453820-a210a2b9-fa40-4f68-bd6c-722467dc75b0.png">
4. have this error when running the code: this.easypostClient.Webhook.validateWebhook is not a function

### What was expected?

because I can see autocomplete suggestion for `validateWebhook` method - I expect to be able to use it : )
please explain how to use this method

### Sample Code

you have this method in `types`: https://github.com/EasyPost/easypost-node/blob/master/types/Webhook/Webhook.d.ts#L104

but do not have it in `services`: https://github.com/EasyPost/easypost-node/blob/master/src/services/webhook_service.js#L8

### Relevant logs

`this.easypostClient.Webhook.validateWebhook is not a function`

**Comment by jchen293** [comment]
Hi @magnafilix

Thanks for submitting this GitHub issue and putting everything together. As part of the V6 rewrite, we have moved the `validateWebhook` function to the `Util` class [here](https://github.com/EasyPost/easypost-node/blob/master/src/utils/util.js#L114). You would need to import the `Util` and do something like this `Util.validateWebhook`, you can find an example of how we used it at the [unit tests](https://github.com/EasyPost/easypost-node/blob/master/test/services/webhook.test.js#L89) we wrote.

I have created a PR #378 to address this issue, let me know if you have more questions.

**Comment by magnafilix** [comment]
hey @jchen293, thanks for a quick response

I see some discussion is going on [here](https://github.com/EasyPost/easypost-node/pull/378#issuecomment-1536428305), is this related to _how to use/import `Util` class_?

**Comment by magnafilix** [comment]
and if yes, should I wait for the next release?

**Comment by jchen293** [comment]
Hi @magnafilix

Adding the typescript file for `Util`will add the type definition of the function in the `Util` class and does not impact the use/import of `Util`. There is no need to wait until the next release to use the `validateWebhook` function.

**Comment by magnafilix** [comment]
I can be annoying I know, but I would really appreciate your help with this question above...

**Comment by magnafilix** [comment]
hey @jchen293, I think this would be the last question, it is about `basic auth`

here in [documentation](https://www.easypost.com/docs/api/node#basic-authentication) it says: _Basic authorization requires that a username and password combination along with the webhook URL be passed during webhook creation._

I created a webhook like this using easypost dashboard `Webhooks and Events` [section](https://www.easypost.com/account/webhooks-and-events) and I was able to receive webhook event, but there is no such header as `Authorization` as it is mentioned here:
<img width="729" alt="Screenshot 2023-05-09 at 11 54 03" src="https://user-images.githubusercontent.com/23529889/237045907-23aacb34-3333-460c-a58b-cbd88d07a1c1.png">

**Comment by Justintime50** [comment]
Hey @magnafilix great question! Since this is a question about our API and not the code of this client library, I'd like to refer you to our awesome support team who will have better context on those items. You can reach them at support@easypost.com and should be able to hook you up with an answer to your question. Best!

**Comment by magnafilix** [comment]
thanks, @Justintime50!

**Comment by jchen293** [comment]
> hey @jchen293, just a quick question regarding import:
> 
> when I do this: `import Util from '@easypost/api/src/utils/util';` I receive this: <img alt="Screenshot 2023-05-08 at 13 10 02" width="1144" src="https://user-images.githubusercontent.com/23529889/236798121-014b99c4-def1-4041-b301-1041bd2f8675.png"> <img alt="Screenshot 2023-05-08 at 13 07 49" width="931" src="https://user-images.githubusercontent.com/23529889/236797156-0b223a26-1e31-42b6-a777-da3f5419e375.png">
> 
> project is written in `typescript`, maybe this is the reason? but I neither can not run the project nor use `Util` because of the error above

@magnafilix We have just released V6.5.0 to address the issues you have brought up. You should able to access the Utility functions in below way:

```node
const client = new EasyPostClient("api_key");
client.Utils.validateWebhook();
```

Thanks again for reporting these issues again😄, and please let us know if you have any other questions

**Comment by magnafilix** [comment]
> > hey @jchen293, just a quick question regarding import:
> > when I do this: `import Util from '@easypost/api/src/utils/util';` I receive this: <img alt="Screenshot 2023-05-08 at 13 10 02" width="1144" src="https://user-images.githubusercontent.com/23529889/236798121-014b99c4-def1-4041-b301-1041bd2f8675.png"> <img alt="Screenshot 2023-05-08 at 13 07 49" width="931" src="https://user-images.githubusercontent.com/23529889/236797156-0b223a26-1e31-42b6-a777-da3f5419e375.png">
> > project is written in `typescript`, maybe this is the reason? but I neither can not run the project nor use `Util` because of the error above
> 
> @magnafilix We have just released V6.5.0 to address the issues you have brought up. You should able to access the Utility functions in below way:
> 
> ```js
> const client = new EasyPostClient("api_key");
> client.Utils.validateWebhook();
> ```
> 
> Thanks again for reporting these issues again😄, and please let us know if you have any other questions

works like a charm, thanks for such a quick release
